go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/grpc-ecosystem/go-grpc-middleware/v2
 go 1.14
 
 require (
-	github.com/gogo/protobuf v1.2.1
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.4.3
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.7.0
 	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
 	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
-	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55 // indirect
-	google.golang.org/grpc v1.28.0
+	google.golang.org/grpc v1.30.1
+	google.golang.org/grpc/examples v0.0.0-20200723182653-9106c3fff523
+	google.golang.org/protobuf v1.25.0 // Github link: https://github.com/protocolbuffers/protobuf-go.
```

Comment:
Is this comment necessary?

Reply:
Up to us

 It's extremely confusing to me when URL is different than GH but somehow project is hosted in GH. 😞

Reply:
I've never seen it before, and if you open that URL in the browser it takes you to`pkg.go.dev` which has a link to the source repository. I'd rather we removed it tbh.